ext/ldap/ldap.stub.php

```diff
@@ -2,265 +2,151 @@
 
 /** @generate-class-entries */
 
+/** @strict-properties */
+final class LDAP
```

Comment:
Yeah, that's also a possibility. I  chose `LDAP` because I thought it's more consistent with other extensions: e.g. `mysqli`, `pdo`. But I've just found out that we now have `FTPConnection`, and we also chose `CurlHandle`. So I'm not sure what to do here. But I'm open to change it to `LDAPConnection`.